packages/common/src/store/ui/toast/types.ts

```diff
@@ -12,6 +14,8 @@ export type Toast = {
   type?: ToastType
   key: string
   timeout?: number | 'MANUAL'
+  leftIcon?: ComponentType<any>
```

Comment:
yes we don't like this change, but we already do it here for the content. instead of making this work also include migrating this off of redux, i think we should do it separately 
https://stackoverflow.com/questions/42128558/storing-react-component-in-a-redux-reducer